nautobot/core/api/views.py

```diff
@@ -467,7 +472,16 @@ def get(self, request):
         nautobot_apps = dict(sorted(nautobot_apps.items()))
 
         # Gather Celery workers
-        workers = celery_app.control.inspect().active()  # list or None
+        try:
+            workers = celery_app.control.inspect().active()  # list or None
+        except redis.exceptions.ConnectionError:
+            # Celery seems to be not smart enough to auto-retry on intermittent failures, so let's do it ourselves:
+            try:
+                workers = celery_app.control.inspect().active()  # list or None
+            except redis.exceptions.ConnectionError as err:
+                logger.error("Repeated ConnectionError from Celery/Redis: %s", err)
+                workers = None
+
```

Comment:
Trying twice seems like an arbitrary number of times.

Reply:
It is, but in the case of the one-off error reported in #6299, it appears sufficient. Do you have an alternative to suggest?

Reply:
Called my bluff there, I have no idea haha